pyproject.toml

```diff
@@ -49,7 +49,7 @@ dev = [
 snow = "snowcli.cli:app"
 
 [tool.hatch.version]
-path = "src/snowcli/__about__.py"
+path = "src/__about__.py"
```

Comment:
There is no such file.
And the checks are failing.
---
This is the case why I think it's a good idea to make PR drafts firstly and then convert them to "ready to review".

Reply:
Am i missing something? As the path looked ok, as it was?

Reply:
Good catch, I was inspired by different fork